README.md

```diff
@@ -25,14 +25,14 @@ We are moving development of Fractal v2.0 and all first-party add-ons into a 'mo
 #### Running locally
 
 1. Clone this repository
-2. Install dependencies - `npm install`
-3. Bootstrap the packages together using Lerna - `npm run bootstrap`
+2. Install dependencies - `yarn install`
```

Comment:
Tbh honest... the scripts don't actually have to use yarn either. Lerna is configured to use yarn via the lerna.json file. Everything else could be changed to plain `npm` as all non-lerna scripts are client-agnostic.